js/util.js

```diff
@@ -40,6 +40,166 @@ function createIdGenerator() {
 
 const isEscapeKey = (event) => event.key === 'Escape';
 
+// Функция для отображения сообщения об ошибке запроса загрузки данных с сервера. пункт 4.2 текущего тз
+// Идея дизайна взята https://us.niemvuilaptrinh.com/article/27-example-of-alert-javascript-for-website
+function alertFailureMesage(message) {
+  document.createElement('div');
+
+  const alertNode = document.createElement('div');
+  alertNode.style.opacity = 1;
+  alertNode.style.position = 'fixed';
+  alertNode.style.height = '100%';
+  alertNode.style.width = '100%';
+  alertNode.style.backgroundColor = 'rgba(94, 110, 141, 0.9)';
+  alertNode.style.left = 0;
+  alertNode.style.top = 0;
+  alertNode.style.zIndex = 100;
+
+  const container = document.createElement('div');
+
+  container.style.position = 'relative';
+  container.style.width = '90%';
+  container.style.maxWidth = '400px';
+  container.style.margin = '4em auto';
+  container.style.background = '#FFF';
+  container.style.color = '#000';
+  container.style.borderRadius = '.25em .25em .4em .4em';
+  container.style.textAlign = 'center';
+  container.style.boxShadow = '0 0 20px rgba(0, 0, 0, 0.2)';
+
+  const messageContainer = document.createElement('p');
+  messageContainer.style.padding = '3em 1em';
+  messageContainer.textContent = message;
+
+  const close = document.createElement('a');
+  close.textContent = 'X';
+  close.addEventListener('click', (event) => {
+    alertNode.remove();
+    event.preventDefault();
+  });
+
+  close.href = '#0';
+
+  close.style.display = 'inline-block';
+  close.style.textIndent = '100%';
+  close.style.position = 'absolute';
+  close.style.color = '#000';
+  close.style.whiteSpace = 'nowrap';
+
+  close.style.top = '8px';
+  close.style.right = '8px';
+  close.style.width = '30px';
+  close.style.height = '30px';
```

Comment:
Выглядит не очень, давай объект совать в close.style

```suggestion
close.style = {
    display: 'inline-block',
    textIndent: '100%',
    ...
}
```